docs/src/free_associative_algebra.md

```diff
@@ -218,3 +218,63 @@ julia> collect(exponent_words(3*b*a*c - b + c + 2))
  []
 ```
 
+### Groebner bases
+
+The function `groebner_basis` provides the computation of a groebner basis of an ideal, given a set of 
+generators of that ideal.
+Since such a groebner basis is not necessarily finite, one can additionally pass a `reduction_bound`
+to the function, to only compute a partial groebner basis.
+
+**Examples**
+
+```jldoctest; setup = :(using AbstractAlgebra)
+
+julia> R, (x, y, u, v, t, s) = FreeAssociativeAlgebra(GF(2), ["x", "y", "u", "v", "t", "s"])
+(Free associative algebra over Finite field F_2 on x, y, u, v, t, s, 
+AbstractAlgebra.Generic.FreeAssAlgElem{AbstractAlgebra.GFElem{Int64}}[x, y, u, v, t, s])
+
+julia> g = Generic.groebner_basis([u*(x*y)^3 + u*(x*y)^2 + u + v, (y*x)^3*t + (y*x)^2*t + t + s])
+5-element Vector{AbstractAlgebra.Generic.FreeAssAlgElem{AbstractAlgebra.GFElem{Int64}}}:
+ u*x*y*x*y*x*y + u*x*y*x*y + u + v
+ y*x*y*x*y*x*t + y*x*y*x*t + t + s
+ u*x*s + v*x*t
+ u*x*y*x*s + v*x*y*x*t
+ u*x*y*x*y*x*s + v*x*y*x*y*x*t
+ ```
+
+ In order to check whether a given element of the algebra is in the ideal generated by a groebner 
+ basis `g`, one can compute its normal form.
+```jldoctest; setup = :(using AbstractAlgebra)
+julia> R, (x, y, u, v, t, s) = FreeAssociativeAlgebra(GF(2), ["x", "y", "u", "v", "t", "s"])
+julia> g = Generic.groebner_basis([u*(x*y)^3 + u*(x*y)^2 + u + v, (y*x)^3*t + (y*x)^2*t + t + s])
+julia> normal_form(u*(x*y)^3*s*t + u*(x*y)^2*s*t +u*s*t + v*s*t, g)
+0
+ ```
+
+ If one has to compute the normal forms of many elements with respect to the same groebner basis, 
```

Comment:
Why is this indented by one space?

Also let's spend "Groebner" consistently in uppercase.